.github/workflows/test-emacs-plugin.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+name: test Emacs plugin
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: Emacs
+    runs-on: ubuntu-latest
+    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v2-wip
+    steps:
+      - name: checkout
+        if: ${{ github.event_name == 'push' }}
+        uses: actions/checkout@v2
+      - name: checkout
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: build
+        run: cmake --build . --config Debug
+      - name: install
+        run: sudo cmake --install . --config Debug
+      - name: test executable
+        run: quick-lint-js --help
```

Comment:
Nit: I wouldn't bother checking that `quick-lint-js` installed. But I'm not super against it.